src/meta/src/hummock/manager/worker.rs

```diff
@@ -150,6 +150,7 @@ where
                 tracing::info!("Cancelled compaction task {}", task_id);
                 sync_point!("AFTER_CANCEL_COMPACTION_TASK_ASYNC");
             }
+            _ => {}
```

Comment:
It might not be a good idea for matching `_` since it won't notice us about new variants in the future. What about making it more explicit?